tests/conftest.py

```diff
@@ -50,6 +50,12 @@ def api(repos_path):
     return pyglottolog.Glottolog(str(repos_path))
 
 
+@pytest.fixture
+def caching_api(repos_path):
+    """Glottolog instance from shared directory for read-only tests."""
+    return pyglottolog.Glottolog(str(repos_path), cache=True)
```

Comment:
Off-topic: since we dropped Python 2 support, we can probably start accepting and passing on `pathlib.Path` instances (besides strings).

Reply:
Ah, I think that's already the case. At some point, `pytest`'s `tmpdir` fixture could not be passed whenever `pathlib.Path` was expected, which is where these casts come from in the tests.

Reply:
Probably good to start replacing `tmpdir` (which stems from `py`) with `tmp_path`.